universal-login-relayer/src/integration/ethereum/DeploymentExecutor.ts

```diff
@@ -24,8 +24,12 @@ export class DeploymentExecutor implements IExecutor<Deployment> {
       const {hash, wait, gasPrice} = transactionResponse;
       ensureNotFalsy(hash, TransactionHashNotFound);
       await this.deploymentRepository.markAsPending(deploymentHash, hash!, gasPrice.toString());
-      await wait();
-      await this.deploymentRepository.setState(deploymentHash, 'Success');
+      const {gasUsed} = await wait();
+      if (gasUsed) {
+        await this.deploymentRepository.markAsSuccess(deploymentHash, gasUsed.toString());
+      } else {
+        throw new Error('No gas used');
```

Comment:
Gas used not found in transaction receipt

Reply:
We should introduce new error or just handle it in else like this.deploymentRepository.markAsError(deploymentHash, 'Gas used not found in transaction receipt)